libobs/CMakeLists.txt

```diff
@@ -2,28 +2,43 @@ cmake_minimum_required(VERSION 3.22...3.25)
 
 legacy_check()
 
+include(cmake/obs-version.cmake)
+
+find_package(Threads REQUIRED)
+
 # cmake-format: off
-add_library(libobs-version STATIC EXCLUDE_FROM_ALL)
-add_library(OBS::libobs-version ALIAS libobs-version)
-# cmake-format: on
-configure_file(obsversion.c.in obsversion.c @ONLY)
-target_sources(libobs-version PRIVATE obsversion.c obsversion.h)
-set_property(TARGET libobs-version PROPERTY FOLDER core)
+if(OS_WINDOWS OR OS_MACOS)
+  set(ffmpeg_version 6)
+else()
+  set(ffmpeg_version 5.1)
```

Comment:
That would indeed be correct - this is adjusted in the PR that enables the non-legacy path for Linux, but could just be backported.

Reply:
I think it's better to just get it right in this PR, since we know that's the reality we currently support, instead of writing one value only to change it in the next PR.